lib/couchdb.js

```diff
@@ -1238,37 +1163,39 @@ CouchDB.prototype._find = function(dbName, query, cb) {
 };
 
 /**
- * Create an index in couchdb,
- * you can specify the ddocName and indexName,
- * regarding how to create model/property index, see discussion in
- * https://github.com/strongloop/loopback-connector-cloudant/pull/153#issuecomment-307177910
- * @param {String} ddocName without prefix `_design/`
- * @param {String} indexName index name
- * @param {Array} fields we only allow single field atm,
- * so it should be a string actually
+ * A model index's naming convertion: '_design/LBModel__Foo__LBIndex__foo_index',
+ * this function returns the model prefix, default as 'LBModel'.
+ * 
+ * TBD: Do we allow customized prefix? same question for 
+ * `getIndexPropertyPrefix`, allow the flexibility would cause 
+ * potential race condition, e.g. two models have same index's 
+ * ddoc name
```

Comment:
Comments removed and create a new issue to keep discussion
https://github.com/strongloop/loopback-connector-couchdb2/issues/22